src/System Application/App/Extension Management/src/ExtensionMarketplace.Codeunit.al

```diff
@@ -487,12 +489,6 @@ codeunit 2501 "Extension Marketplace"
     [EventSubscriber(ObjectType::Codeunit, Codeunit::"System Action Triggers", OpenAppSourceMarket, '', false, false)]
     local procedure OpenAppSourceMarket()
     begin
-#if not CLEAN24        
-#pragma warning disable AL0432
-        Page.Run(Page::"Extension Marketplace");
-#pragma warning restore AL0432
-#else        
-        Hyperlink('https://aka.ms/BCAppSource');
-#endif        
+        Page.Run(2515); // Using ID to avoid circular dependency.
```

Comment:
same... this subscriber should be placed in the other module

Reply:
Good one - thanks